testUtils/httpsMocker.js

```diff
@@ -13,11 +14,21 @@ export const HttpsScenarioBuilder = (() => {
     nextResponse.push(response);
   };
 
-  const getNextResponse = () => {
+  const getNextData = () => {
     const response = nextResponse.pop();
     return response ? response : defaultResponse;
   };
 
+  const isRequestShouldFinish = () => {
+    if (isNextRequestShouldFinish) return true;
```

Comment:
Shouldn't the value of `isNextRequestShouldFinish` be updated?

Reply:
No, the default state of isNextRequestShouldFinish is `true` so we don't want to change it.
if its false we get this back to a `true`